doto-backend/test/task.test.js

```diff
@@ -11,6 +11,24 @@ const validUser = new UserModel({
     themePreference: 'dark'
 })
 
+const invalidUser = new UserModel({
```

Comment:
invalidUser is not currently being used. It's probably better if we delete it for now.